test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/https/WorkbenchHttpsIntegrationTest.java

```diff
@@ -180,10 +189,10 @@ private SSLConnectionSocketFactory getSSLConnectionSocketFactory() {
             builder.loadTrustMaterial(null, new TrustAllStrategy());
             sslsf = new SSLConnectionSocketFactory(
                     builder.build(), SSLConnectionSocketFactory.ALLOW_ALL_HOSTNAME_VERIFIER);
+
+            return sslsf;
         } catch (Exception e) {
-            Assertions.fail("Error in SSL setup", e);
+            throw new RuntimeException("Error in SSL setup", e);
```

Comment:
Some portions of this test throw exceptions, some use failing assertions instead. Is there any reason for this discrepancy?

(This is not necessarily bad - I'm just curious what the reasoning was.)